packages/components/psammead-media-player/src/Canonical/index.jsx

```diff
@@ -1,38 +1,63 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
 import { string } from 'prop-types';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
+import Guidance from '../Guidance';
+import useTimeout from './useTimeout';
 
-const Canonical = ({ src, title, placeholderSrc }) => {
-  const backgroundStyle = `
-    background-image: url(${placeholderSrc});
-    background-repeat: no-repeat;
-    background-size: contain;
-  `;
+const TIMEOUT_MS = 5000;
```

Comment:
Is it worth having a default timeout in Psammead and then having this override-able via a prop passed in from Simorgh?

This way different page types that integrate with the media player could specify different timeout periods e.g. an article may want a shorter timeout period than a MAP asset.

Reply:
This is a great idea, Chris. Initially I didn't want to complicate the API further, but with a sensible default and option to override it needn't!

Reply:
I've made these changes, @hindsc52. 👍